spec2vec/SpectrumDocument.py

```diff
@@ -2,7 +2,24 @@
 
 
 class SpectrumDocument(Document):
+    """Create documents from spectra.
+
+    Every peak (and loss) positions (m/z value) will be converted into a string "word".
+    The entire list of all peak words forms a spectrum document. Peak words have
+    the form "peak@100.3" (for n_decimals=1), and losses have the format "loss@100.3".
+    """
     def __init__(self, spectrum, n_decimals=1):
+        """
+
+        Parameters
+        ----------
+        spectrum : matchms SpectrumType
+            Input spectrum.
+        n_decimals : int, optional
+            Peak positions are converted to strings with n_decimal decimals.
+            The default is 1, which would convert a peak at 100.381 into the
+            word "peak@100.4".
```

Comment:
What happens to multiple peaks with m/z within that precision for example 100.43 and 100.41? Will the peak with the highest intensity be kept and rest discarded?

Reply:
In that case the document would simply contain two words "peak@100.4" with each the respective intensity of the original peak. Keeping only the highest intensity peak would probably also work, but simply leaving all peaks in seemed cleaner to me (cleaner as in making less assumptions).